blog/content/post/2021-02-21-tigerlake-osf.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Open Source Firmware on TigerLake platforms
+abstract: 'This post describes efforts of building open source firmware for
+           Tiger Lake UP3 RVP platform and the problems faced in the process.
+           Tiger Lake is one of the newest Intel processors for which the FSP
+           and EDK2 MinPlatform has been recently released.'
+cover: /covers/tigerlake.jpg
+author: michal.zygowski
+layout: post
+published: true
+date: 2021-02-21
+archives: "2021"
+
+tags:
+  - Firmware
+categories:
+  - Firmware
+
+---
+
+# Introduction
+
+If somebody would tell 6 years ago that Intel will support open source
+firmware, he would be laughed at instantly. If we recall time, like 15 years
+ago where the datasheets were more open and were sufficient to write open
+source firmware, today it is not possible. Silicon vendors are hiding the
+intellectual property contained in the processors. It would seem like the open
+source firmware is doomed, but...
+
+Thankfully there are companies and Intel employees that try to make impact and
+change this situation. For example Google supporting the coreboot project on
+their Chromebooks encourage Intel to release the Firmware Support Package
+([FSP](https://www.intel.com/content/www/us/en/intelligent-systems/intel-firmware-support-package/intel-fsp-overview.html)).
+The FSP is a bundled silicon initialization code in a binary form with well
+documented interface and configuration options. It simplifies new hardware
+enabling and reduces cost of overall firmware development. While it doesn't
+solve all problems and sometimes causes issues, kudos should go to Intel for
+supporting the open source firmware. Special credits should go to the open
+source representatives from Intel like Nataniel DeSimone, Vincent Zimmer, Brian
+Richardson and Isaac Oram. The are often present on various open source
```

Comment:
@miczyg1 I would also link their LinkedIn profiles

Reply:
Done, but it seems Nate does not have one